kansha/board/boardsmanager.py

```diff
@@ -59,15 +60,23 @@ def _get_board(self, data_board):
         return self._services(Board, data_board.id, self.app_title, self.app_banner, self.theme,
                               self.card_extensions, self.search_engine)
 
+    def create_template_empty(self):
+        '''Get a default empty template'''
+        board = DataBoard(title=u'Empty board', is_template=True)
```

Comment:
Important: a component can use the data part of an other component. Component uses an other component thru its public API exposed from its logic part. So, here, create a `Board`, not a `DataBoard`